book/src/dev/release-process.md

```diff
@@ -0,0 +1,117 @@
+
+Zebra version numbers have three parts: `major.minor.patch`.
+For example, version `3.1.11` indicates major version 3, minor version 1, and patch level 11.
+
+The version number is incremented based on the level of change included in the release.
+
+<div class="alert pre-release">
+
+**NOTE**: <br />
+As Zebra is in a `pre-release` state (is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version).
+The pre-release version is denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version.
+
+</div>
+
+| Level of change | Details |
+|:---             |:---     |
+| Major release   | Contains significant new features, and commonly correspond to network upgrades; some technical assistance may be needed during the update. When updating to a major release, you may need to follow the specific upgrade instructions provided in the release notes. |
+| Minor release   | Contains new smaller features. Minor releases are should be fully backward-compatible. No technical assistance is expected during update. |
+| Patch release   | Low risk, bug fix release. No technical assistance is expected during update. |
+
+<a id="supported-releases"></a>
+
+### Supported Releases
+
+Older Zebra versions are always supported until:
+* the next Zebra major, minor or patch release.
+
+Initially, we can only guarantee support for the latest Zebra release. We might be able to provide support for earlier releases, or we might ask you to upgrade to the latest release.
+
+Our support periods will be extended as we gain experience with supporting Zebra releases.
+
+Older Zebra versions that only support previous network upgrades will never be supported, because they are operating on an unsupported Zcash chain fork.
+
+<a id="updating"></a>
+
+### Supported update paths
+
+You can update to any version of Zebra, provided that the following criteria are met:
+
+* The version you want to update *to* is supported.
+* The version you want to update *from* is within one major version of the version you want to
+    upgrade to.
+
+See [Keeping Up-to-Date](guide/updating "Updating your projects") for more information about updating your Zebra projects to the most recent version.
+
+<a id="previews"></a>
+
+### Preview releases
+
+We let you preview what's coming by providing Release Candidate \(`rc`\) pre-releases for some major releases:
+
+| Pre-release type  | Details |
+|:---               |:---     |
+| Beta              | The release that is under active development and testing. The beta release is indicated by a release tag appended with the `-beta` identifier, such as  `8.1.0-beta.0`.      |
+| Release candidate | A release that is feature complete and in final testing. A release candidate is indicated by a release tag appended with the `-rc` identifier, such as version `8.1.0-rc.0`. |
```

Comment:
I think I originally said "feature-complete", but the team decided not to say it that way in yesterday's meeting.

```suggestion
| Release candidate | A release for final testing of new features. A release candidate is indicated by a release tag appended with the `-rc` identifier, such as version `8.1.0-rc.0`. |
```